ChangingArray.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW_4
+{
+    public class ChangingArray
+    {
+        public int LetterArray(int[] array, string meaning)
+        {
+            Console.WriteLine();
+            Console.WriteLine(new string('-', 50));
+
+            switch (meaning)
+            {
+                case "evvenArray":
+                    Console.WriteLine("Масив четных чисел в буквах");
+                    break;
+                case "oddArray":
+                    Console.WriteLine("Масив не четных чисел в буквах");
+                    break;
+            }
+
+            string[] arrayA = new string[array.Length];
+            int index = 0;
+            for (int i = 0; i < array.Length; i++)
+            {
+                switch (array[i])
+                {
+                    case 1:
```

Comment:
Do we really need this whole checking structure? 
If yes, please create separate variables for the checking values, so the code can be reused later on.
If no, rework the structure to avoid such gigantic checks